client/src/commands/createSmartContractProjectCommand.ts

```diff
@@ -15,9 +15,10 @@
 import { window, Uri, commands } from 'vscode';
 import { VSCodeOutputAdapter } from '../logging/VSCodeOutputAdapter';
 import { CommandUtil } from '../util/CommandUtil';
+import * as child_process from 'child_process';
 
-export async function createFabricProject(): Promise<void> {
-    console.log('create Fabric Project');
+export async function createSmartContractProject(): Promise<void> {
+    console.log('create Smart Contract Project');
```

Comment:
Should console logs be left in, or is there some hyperlegendary vscode logging doins available?

Reply:
I'd like to leave this in, it's useful for debugging purposes and not visible to the user